### DISCUSSION TITLE
Managed Satellites Customisation

### DISCUSSION THREAD

**Discussion by fifise77** [JarvyJ/HomeIntent#523] [Support Questions] [answered]
Hello, 

First thank for your hard work and this amazing app.
I am trying to use it with french seapking language

is there a way to managed the Satellite to use "arecord" as a audio recording instead of "PyAudio" ?

**Comment by JarvyJ** [comment, marked as answer]
Yes! There are actually two ways you could go about this.

The easiest is to just switch it from a managed satellite to a manual one. It would mostly involve removing the `managed_satellite` configs in `config.yaml` and then adding the satellite name in the `satellite_ids` section instead. From there, you can just go to the satellite's Rhasspy instance and change what you want, but it will at least initially have all the settings correct because it was managed.

**Reply by JarvyJ** [reply]
The other way would be to modify the satellite's Rhasspy config, go to the Rhasspy "advanced" page and copy the json blob, and then save it in the config folder as `satellite_profile.json` and the code should load that file instead. This of course would set _all_ satellite's with that config, which may or may not be what you want.

**Reply by fifise77** [reply]
Hello,
I switch to manual once the satellite got all the initial setting set. And it work great.

Do you plan to add the possility to modify some setting throught the manage option ?

**Reply by JarvyJ** [reply]
Yeah! The features are just slowing trickling in, but I am planning to add some customisability to managed satellites.
